core/genesis.go

```diff
@@ -375,8 +375,8 @@ func DefaultGardenGenesisBlock(consensusEngine string) *Genesis {
 		return &Genesis{
 			Config:     params.Blake3PowGardenChainConfig,
 			Nonce:      66,
-			ExtraData:  hexutil.MustDecode("0x11bbe8db4e347b4e8c937c1c8370e4b5ed33adb3db69cbdb7a38e1e50b1b82fb"),
-			GasLimit:   5000000,
+			ExtraData:  hexutil.MustDecode("0x11bbe8db4e347b4e8c937c1c8370e4b5ed33adb3db69cbdb7a38e1e50b1b82fa"),
```

Comment:
Did you mean to change just one letter of the hash?

Reply:
Interesting!

Reply:
yes, we were getting same hash for 2 different networs because the config was the same, so changed it to update the hash